cmd/micro/initlua.go

```diff
@@ -136,6 +136,7 @@ func luaImportMicroBuffer() *lua.LTable {
 	ulua.L.SetField(pkg, "ByteOffset", luar.New(ulua.L, buffer.ByteOffset))
 	ulua.L.SetField(pkg, "Log", luar.New(ulua.L, buffer.WriteLog))
 	ulua.L.SetField(pkg, "LogBuf", luar.New(ulua.L, buffer.GetLogBuf))
+	ulua.L.SetField(pkg, "NewCursor", luar.New(ulua.L, buffer.NewCursor))
```

Comment:
It's better to move it above, after `NewBufferFromFile`, to keep these `New*` functions grouped together.